mne/epochs.py

```diff
@@ -2437,32 +2438,15 @@ def bootstrap(epochs, random_state=None):
 
 def _check_merge_epochs(epochs_list):
     """Aux function"""
-    event_ids = set(tuple(epochs.event_id.items()) for epochs in epochs_list)
-    if len(event_ids) == 1:
-        event_id = dict(event_ids.pop())
-    else:
+    if len(set(tuple(epochs.event_id.items()) for epochs in epochs_list)) != 1:
         raise NotImplementedError("Epochs with unequal values for event_id")
```

Comment:
@dengemann you okay with these changes? I don't think the actual values for `event_ids`, `baseline`, `tmin`, and `tmax` need to be returned by the function. It's checking to make sure they're identical, so the values in the `epochs.copy()` below should already have the correct ones. Make sense?